src/js/hora.js

```diff
@@ -41,6 +41,13 @@ define([
             }
         };
 
+        Hora.event = {
```

Comment:
[nitpick] We can add this in the initial Hora definition:

var Hora = {
    isDebug: false
    events: {
        // ...
    }
};